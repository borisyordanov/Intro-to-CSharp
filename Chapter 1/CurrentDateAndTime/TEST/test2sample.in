1981
6
24
15
1
43

