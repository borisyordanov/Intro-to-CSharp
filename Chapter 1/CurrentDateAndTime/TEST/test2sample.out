6/24/1981 3:01:43 PM
